app/sessionStore.js

```diff
@@ -823,6 +824,18 @@ module.exports.runPreMigrations = (data) => {
   }
 
   if (data.lastAppVersion) {
+    try {
+      // with version 0.22.13, any file downloaded (including the update itself) would get
+      // quarantined on macOS (per work done with https://github.com/brave/muon/pull/484)
+      // this functionality was then reverted with https://github.com/brave/muon/pull/570
+      //
+      // To fix the executable, we need to manually un-quarantine the Brave executable so that it works as expected
+      if (process.platform === 'darwin' && compareVersions(data.lastAppVersion, '0.22.13') === 0) {
+        execSync('xattr -d com.apple.quarantine /Applications/Brave.app')
```

Comment:
`'xattr -d com.apple.quarantine /Applications/Brave.app || true`. In cases where the `.app` doesn't have a quarantine flag `xattr -d` might fail.